src/Symfony/Bundle/FrameworkBundle/Resources/config/mailer.xml

```diff
@@ -6,23 +6,23 @@
 
     <services>
         <service id="mailer.mailer" class="Symfony\Component\Mailer\Mailer">
-            <argument type="service" id="mailer.transport" />
+            <argument type="service" id="mailer.default_transport" />
             <argument type="service" id="messenger.default_bus" on-invalid="ignore" />
         </service>
         <service id="mailer" alias="mailer.mailer" />
         <service id="Symfony\Component\Mailer\MailerInterface" alias="mailer.mailer" />
 
-        <service id="mailer.transport" class="Symfony\Component\Mailer\Transport\TransportInterface">
+        <service id="mailer.default_transport" class="Symfony\Component\Mailer\Transport\TransportInterface">
```

Comment:
changing the id here might cause trouble for developers who have already updated and are using this id